alts/shared/models.py

```diff
@@ -99,7 +99,7 @@ class CeleryConfig(BaseModel):
     task_tracking_timeout: int = 3600
     # Supported architectures and distributions
     supported_architectures: typing.List[str] = ['x86_64', 'i686', 'amd64',
```

Comment:
Why do you not move those items to constants.py?

Reply:
Good catch, fixed